### DISCUSSION TITLE
Subtitles language

### DISCUSSION THREAD

**Discussion by Jupiteur** [pystardust/ani-cli#684] [Ideas] [answered]
We could add a language option for the subtitles

**Comment by port19x** [comment, marked as answer]
In case you mean sub languages other than english, thats close enough to impossible to not elaborate further.

If you mean a --sub or --dub option for auto-selecting the respective version of an anime, that has proven to be inconsistent in the past

**Reply by Jupiteur** [reply]
And what about scrapping other streaming websites providing different languages? I believe there was a discussion about choosing between multiple websites

**Reply by CoolnsX** [reply]
there is a provider in allanime that provides different lang subtitles however the occurence of that provider is low, and probability is more on recent anime so..

**Reply by CoolnsX** [reply]
https://raw.githubusercontent.com/CoolnsX/ani-cli-status/main/analytics.png

the "vrv" one is multi lang one but its probability is low compared to others

**Reply by Jupiteur** [reply]
We could build a database of providers with data about supported languages (for instance neko-sama.fr provides 100% french sub and some dub) and an associated scrapping function
